web-server/sgx-wallet-impl/src/schema/entities.rs

```diff
@@ -90,6 +92,7 @@ pub struct WalletStorable {
 
     pub owner_name: String,
     pub phone_number: Option<String>,
+    pub otp_phone_number: Option<String>,
```

Comment:
When user creates a wallet for the first time, the OTP phone number should default to the user's phone number.